bigquery_etl/shredder/config.py

```diff
@@ -355,3 +356,26 @@ def find_glean_targets(pool, client, project=SHARED_PROD):
             if any(field.name == CLIENT_ID for field in table.schema)
         },
     }
+
+
+EXPERIMENT_ANALYSIS = "moz-fx-data-experiments"
+
+
+def find_experiment_analysis_targets(pool, client, project=EXPERIMENT_ANALYSIS):
+    """Return a dict like DELETE_TARGETS for experiment analysis tables."""
+    datasets = {dataset.dataset_id for dataset in client.list_datasets(project)}
+
+    tables = [
+        table
+        for tables in pool.map(
+            client.list_tables,
+            [bigquery.DatasetReference(project, dataset_id) for dataset_id in datasets],
+        )
```

Comment:
There's no need to construct the dataset references here, they are available via `DatasetListItem.reference`. Additionally, setting `chunksize=1` ensures that each thread in the pool only claims one dataset at a time to work on, which is ideal for small workloads like this.
```suggestion
    datasets = {dataset.reference for dataset in client.list_datasets(project)}

    tables = [
        table
        for tables in pool.map(
            client.list_tables,
            datasets,
            chunksize=1,
        )
```